tests/shardddl3/conf/source1.yaml

```diff
@@ -1,7 +1,7 @@
 source-id: mysql-replica-01
 flavor: ''
 enable-gtid: false
-enable-relay: false
+enable-relay: true
```

Comment:
How about enabling GTID only for one source? (and keep another one use remote binlog).